scripts/zones/Misareaux_Coast/mobs/Gration.lua

```diff
@@ -29,13 +29,9 @@ end
 entity.onMobFight = function(mob, target)
     local enmityList = mob:getEnmityList()
     for _, v in ipairs(enmityList) do
-        local shouldintimidate = math.random(1, 20)
-        if shouldintimidate >= 19 then
-            if v.entity:hasStatusEffect(xi.effect.INTIMIDATE) then
-                v.entity:delStatusEffectSilent(xi.effect.INTIMIDATE)
-            end
-
-            v.entity:addStatusEffectEx(xi.effect.INTIMIDATE, 0, 1, 0, 1)
+        -- repeatedly apply short duration intimidate (5 sec) with 50% power to all entities on enmity list
+        if not v["entity"]:hasStatusEffect(xi.effect.INTIMIDATE) then
```

Comment:
Does accessing entity work this way? I've never used this kind of styling in lua

Reply:
> Does accessing entity work this way? I've never used this kind of styling in lua

Yep, "Lua supports this representation by providing a.name as syntactic sugar for a["name"]." from lua guide.